README.md

```diff
@@ -8,12 +8,4 @@ Welcome to the Futured Dev Academy!
 - Radek Doležal, iOS developer, <radek.dolezal@futured.app>
 - Mikoláš Stuchlík, iOS developer, <mikolas.stuchlik@futured.app>
 - Honza Pollak, iOS developer, <jan.pollak@futured.app>
-- Marie Re, iOS developer, <marie.re@futured.app>
-
-## Configuration management
-
-### Tools
-
-- Language: **Swift 5**
-- IDE: **Xcode 14**
-- Dependency management: **[Swift package manager](https://swift.org/package-manager/)**
\ No newline at end of file
+- Marie Re, iOS developer, <marie.re@futured.app>
```

Comment:
Nebylo to pred tim s tim dlouhym popiskem k nicemu lepsi?

Reply:
Ok, tak ja to prcam.